plugins/com.google.cloud.tools.eclipse.util/templates/appengine/pom.xml.flex.spring.boot.ftl

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+  <version>${projectVersion}</version>
+
+  <groupId>${projectGroupId}</groupId>
+  <artifactId>${projectArtifactId}</artifactId>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.showDeprecation>true</maven.compiler.showDeprecation>
```

Comment:
Do we really want to show deprecation?
In general I prefer to keep samples minimal.

Reply:
The original and current standard env `pom.xml` had this, and this is currently in every `pom.xml` we generate. I'll remove all of them here. Just let me know if you change your mind.